pyproject.toml

```diff
@@ -4,7 +4,7 @@
 [project]
 name = "postgresql-charms-single-kernel"
 description = "Shared and reusable code for PostgreSQL-related charms"
-version = "16.1.0"
+version = "16.1.1"
```

Comment:
Shouldn't we bump it to 16.2.0 instead? Those prefixes helpers seems to be a new feature.

Reply:
My thinking was that changes are additive, so they shouldn't break anything.